src/support_sphere_db/src/ssdb/respository/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ssdb.respository.user_repository import UserRepository
+
+__all__ = ['UserRepository']
```

Comment:
What is this `Repository` package? it's for db interaction it seems like?

Also, `repository` sub-package is misspelled.

Reply:
Yes, it is for db interaction. It is  based on the simplified [repository pattern](https://python.plainenglish.io/design-patterns-in-python-repository-pattern-1c2e5070a01c)

Ah, thanks! Updated the misspelling.